### PR TITLE
Update README.md (AO-16692)

### PR DESCRIPTION
**Pull request by d-maslyk**
* AO-16692
* Readme changes - Documentation update - to be aligned with AppOptics doc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,26 +93,50 @@ services:
 
 #### Kubernetes
 
-If you wanted to run this on Kubernetes as a sidecar for monitoring specific services, you can follow the instructions below which use Zookeeper as an example.
+If you wanted to run this on Kubernetes as a sidecar for monitoring specific services, you can follow the instructions below which use Apache Server as an example. This setup the agent will monitor only services running in particular pod(s), not Kubernetes itself.
```
Comment (tomekkolo) [inline README.md:96]: maybe: 
In this setup, the agent will monitor

File: README.md [hunk 2]
```diff
@@ -171,7 +202,8 @@ index 294c4b4..babff7d 100644
            hostPath:
              path: /proc
 ```
-Notice that we're not utilizing [Environment Parameters](###environment-parameters) to turn on Kubernetes plugin. After editing deployment manifest it's time to create it - follow the steps in [Installation](##installation).
+Notice that we're not utilizing [Environment Parameters](#environment-parameters) to turn on Kubernetes plugin. When you're attaching taskfiles and plugin configuration files through configMaps, there's 
+no need to set environment variables `SWISNAP_ENABLE_<plugin-name>`. SolarWinds Snap Agent will automatically load plugins based on files stored in configMaps.
```
Comment (tomekkolo) [inline README.md:206]: not needed new line ?

File: README.md (hunk 1 above)
Comment (d-maslyk) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (d-maslyk) [reply README.md]: Fixed. Thanks.

**Pull request merged**